src/data_aggregator.ts

```diff
@@ -302,36 +191,13 @@ export class DataAggregator {
 
   async currentPrice(): Promise<BigNumber> {
     switch (this.config.aggregationMethod) {
-      case AggregationMethod.TRADES:
-        return this.currentPriceFromTrades()
       case AggregationMethod.MIDPRICES:
```

Comment:
Unless we anticipate adding another aggregation method sometime soon, I think we should get rid of the AggregationMethod config entirely.

Reply:
I think we might be revisiting how we aggregate the prices soon, so I'd leave this for the time being. I'd be happy to clean it up later if we end up not using it.